lib/pbench/test/unit/server/database/__init__.py

```diff
@@ -0,0 +1,351 @@
+"""The SQLAlchemy mock infrastructure here is not completely generalized, but
+is currently sufficient to support the ServerConfig and Audit DB unit tests.
+"""
+
+import operator
+from typing import Optional
+
+from sqlalchemy import Column
+from sqlalchemy.util import immutabledict
+
+from pbench.server.database.database import Database
+
+
+def dumpem(object: object, depth: int = 0, max_depth: int = 20):
```

Comment:
I don't see it being used anywhere, should we comment out this function if this is only for archival purpose?

Reply:
I think it's a bad idea to have commented-out code in the source -- it should either be "live" or we should delete it.  (I'm OK with keeping this around, although I'm wondering whether this is really specific to SQLAlchemy or whether we should toss it into a utility module for general use.)

Reply:
Of course it's not technically specific to SQLAlchemy: it's a way to explore classes through unit tests by looking at the actual live structure and values. The point was that if it's in the code we can find it and re-use it as necessary, but it's certainly not something we want in our product code. i didn't see any point in trying to figure out where to move it.